README.md

```diff
@@ -48,18 +48,26 @@ The following is an example of a valid ToMaR:
     "mqttTopic": "devices/([^/]+)/data/(\b(all|new|old)\b)",
      "kafkaTopic": "device_$1_data",
      "kafkaKey": "device_$2"
+  },
+  {
+      "mqttTopic": "locations/([^/]+)(?:\\/.*)?$",
+      "kafkaTopic": "locations",
+      "kafkaKey": "location_$1"
   }
 ]
 ```
 
-- The expression `.*` is used to represent the wildcard `#`, which in turn represents one or more levels in the MQTT topic hierarchy.
-However, there are some cases that can lead to unexpected behavior when using the `.*` wildcard.
+The expressions `.*` and `(?:\\/.*)?$` are used to represent the wildcard `#`, which in turn represents one or more levels in the MQTT topic hierarchy.
```

Comment:
I think a user can understand `.*` but can you provide more details about the `(?:\\/.*)?$` to explain why and how it's different from `.*`. Why the user should use one or the other?

Reply:
I explained down below.

Reply:
Tbh you are comparing the two with different examples, it doesn't help the user. See my comment below.